### DISCUSSION TITLE
Static IP detaction from IPAM

### DISCUSSION THREAD

**Discussion by PG-Sysad** [netbox-community/netbox#13742] [Q&A]
Hey im new to Netbox,

me and my Team used the Microsoft IPAM Server. We would like to swtich here. Does the IPAM from Netbox detect static IP Adresses automatically? 
Maybe by dns or something?

**Comment by adparis99** [comment]
NetBox, on its own, doesn't do anything automatically. With some combination of scripts ([Python](https://github.com/netbox-community/pynetbox), [Ansible](https://docs.ansible.com/ansible/latest/collections/netbox/netbox/index.html#modules), [NetBox Custom Scripts](https://docs.netbox.dev/en/stable/customization/custom-scripts/)), [webhooks](https://docs.netbox.dev/en/stable/integrations/webhooks/), and [plugins](https://github.com/topics/netbox-plugin), it can do a lot. Depending on how comfortable you are your team are with scripting (and how specific your requirements are), that might the preferred way to go. There are also many publicly available plugins that extend functionality in certain ways. You can search through those and see if any of them meet your needs.

**Reply by PG-Sysad** [reply]
Thank you for your quick reply!
